components/atom/label/Label.tsx

```
@@ -0,0 +1,19 @@
+
+export const Label = ({ children, className, ...props }: LabelProps) => {
+  return (
+    <label
+      className={cn(
+        'min-h-8 py-2 px-1 text-sm leading-4 text-black',
+        className
+      )}
+      {...props}
+    >
+      {children}
+    </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)